src/confd/src/ietf-keystore.c

```diff
@@ -118,8 +118,10 @@ static int change_cb(sr_session_ctx_t *session, uint32_t sub_id, const char *mod
 					goto next;
 				}
 			next:
-				unlink(SSH_PRIVATE_KEY);
-				unlink(SSH_PUBLIC_KEY);
+				if(fexist(SSH_PRIVATE_KEY))
+					unlink(SSH_PRIVATE_KEY);
+				if (fexist(SSH_PUBLIC_KEY))
+					unlink(SSH_PUBLIC_KEY);
```

Comment:
Since we don't check the return value of `unlink()` the new calls to `fexist()` is just waste.  

A better change may be to replace them with:

```c
if (erase(SSH_PRIVATE_KEY))
    ERRNO("Failed removing SSH server private key");
if (erase(SSH_PUBLIC_KEY))
    ERRNO("Failed removing SSH server private key");
```

The `ERRNO()` macro appends a human-friendly error message (mostly directed to us for troubleshooting).  We can rely on `erase()` to skip `ENOENT` errors (file does not exist), which we don't care about in this operation.